test/amd.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit AMD Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+</head>
+<body>
+	<div id="qunit"></div>
+	<script>
+	var beginData,
+		asyncDelay = 1000,
+		times = {
+			pageLoaded: 0,
+			runStarted: 0
+		};
+
+	(function() {
+		var now = Date.now || function() {
+			return new Date().getTime();
+		};
+
+		// Simulate a delay in loading the tests, as when they are loaded
+		// asynchronously using requirejs, steal, etc.
+		window.define = function() {};
+		window.define.amd = true;
+		setTimeout(function() {
+			var script = document.createElement( "script" );
+			script.src = "../dist/qunit.js";
+			script.onload = function loadTests() {
+				QUnit.load = (function( _load ) {
+					times.pageLoaded = now();
+					_load.call( QUnit );
+				})( QUnit.load );
```

Comment:
Oh, good catch. This is actually just copied from the existing `autostart.html` file, so we'll need to update that as well.